test/account_api_test.rb

```diff
@@ -0,0 +1,278 @@
+require "test_helper"
+require "gds_api/account_api"
+
+describe GdsApi::AccountApi do
+  include PactTest
+
+  let(:api_client) { GdsApi::AccountApi.new(account_api_host) }
+
+  let(:authenticated_headers) { { "GOVUK-Account-Session" => govuk_account_session } }
+  let(:govuk_account_session) { "logged-in-user-session" }
+
+  describe "getting a sign-in URL" do
+    let(:redirect_path) { nil }
+
+    before do
+      account_api
+        .upon_receiving("a sign-in request")
```

Comment:
These lines seem to just be human readable strings that don't repeat or match anywhere.
So what are they used for? Are these just Pact versions of `describe` blocks for test stack traces?

Reply:
I think so, yes.  The `given`s correspond to set-up blocks in account-api, but these `upon_receiving`s are descriptive.